README.md

```diff
@@ -65,8 +65,56 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
 
 
 **CA Cluster Issuer Versions:**
-- Download latest version: [CA Cluster - v0.0.1]()  
+- Download latest version: [CA Cluster - v0.0.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=ca-clusterissuer/v0.0.1/ca-clusterissuer.yml)  
 
 **CA Cluster Issuer Sample values.yaml**
 - Download latest sample values.yaml: [values.yaml for
-  v0.0.1](caclusterissuer/values.yml)
+  v0.0.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=ca-clusterissuer/v0.0.1/ca-clusterissuer-data-values.yml)
+
+
+## Cloud Native Registry
+
+</br>
+</br>
+<p align="left">
+  <img src="harbor-logo.png" width="250" title="Harbor Logo">
+</p>
+</br>
+Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency. Harbor supports replication of images between registries, and also offers advanced security features such as user management, access control and activity auditing.
+</br>
+</br>
+
+- The [contour package](#contour) is a prerequisite for Harbor, so that must be installed
+  first. 
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
```

Comment:
https://docs-staging.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-EB11DFDE-6ED4-4864-9E60-669F50D64397.html 

Maybe we could refer to this page instead. This is more detailed and specific to harbor and contour.